stable/osg-hosted-ce/osg-hosted-ce/values.yaml

```
@@ -103,6 +103,10 @@ VoRemoteUserMapping:
 #DnRemoteUserMapping:
 #  - "/DC=foo/DC=bar/OU=Organic Units/OU=Users/CN=YourUserName": osg01
 
+# SCITOKENS url to user mapping
+# SciTokenRemoteUserMapping:
+#   - "https://scitokens.org/osg-connect": osg01
```

```suggestion
    # https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/OSG.yaml
    # - "https://scitokens.org/osg-connect": osg01
    # https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/GLOW.yaml
    # - "https://chtc.cs.wisc.edu": osg02
    # https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/CMS.yaml
    # - "https://cms-auth.web.cern.ch/": osg04
    # https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/ATLAS.yaml
    # - "https://atlas-auth.web.cern.ch/": osg09
```

Fleshing out the commented issuers